scripts/lingui-extract-with-plugins

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
We should really find a nice place to document our future plans. We want to move away from bash scripts as we already have node in place and they are basically untestable. No need to change here, but would be cool if we all could keep that in mind for the next time.

Reply:
I can rewrite it fairly easily. In this case I thought bash was a good choice because I had to perform so many shell commands